Library/Homebrew/cask/spec/cask/artifact/binary_spec.rb

```diff
@@ -20,7 +20,7 @@
     shutup do
       Hbc::Artifact::Binary.new(cask).install_phase
     end
-    expect(FileHelper.valid_alias?(expected_path)).to be true
+    expect(expected_path).to be_a_valid_symlink
```

Comment:
Personally: yes. This PR is a refactoring for slightly shortening two lines of code which ends up adding more lines than it removes.